### PR TITLE
optimizing naming

### PR DESCRIPTION
**Pull request by pengzhengfa**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: naming/src/main/java/com/alibaba/nacos/naming/controllers/ServiceController.java [hunk 1]
```diff
@@ -320,7 +320,7 @@ public String serviceStatus(HttpServletRequest request) throws Exception {
                         Loggers.SRV_LOG.debug("checksum of {} is not consistent, remote: {}, checksum: {}, local: {}",
                             serviceName, serverIp, checksum, service.getChecksum());
                     }
-                    serviceManager.addUpdatedService2Queue(checksums.namespaceId, serviceName, serverIp, checksum);
+                    serviceManager.addUpdatedServiceQueue(checksums.namespaceId, serviceName, serverIp, checksum);
```
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/controllers/ServiceController.java:323]: I think we can change `2` to `To`.If remove this word, the meaning of method will be changed.

Comment (pengzhengfa) [reply naming/src/main/java/com/alibaba/nacos/naming/controllers/ServiceController.java:323]: OK, I'll change it

**Comment by KomachiSion** [comment]
@pengzhengfa please sign cla.

**Comment by pengzhengfa** [comment]
@KomachiSion Hello, Pdf can't be downloaded. It has been sent to email in the form of word document opensource@alibaba-inc.com

**Comment by KomachiSion** [comment]
> @KomachiSion Hello, Pdf can't be downloaded. It has been sent to email in the form of word document [opensource@alibaba-inc.com](mailto:opensource@alibaba-inc.com)

It seems that your git username is not your github user, you can try to change your nacos repo's username and email as your github user.
```
cd to nacos dir

git config username ${github.username}
git config email ${github.email}

```
then re-commit your change and entry [Contributor License Agreement](https://cla-assistant.io/alibaba/nacos?pullRequest=2641) sign cla.

**Comment by pengzhengfa** [comment]
@KomachiSion Hello, still no    
image

**Comment by KomachiSion** [comment]
No, I mean that you config your local git username as your github username, and re-commit you change.

CLA just click sign in Github user and agree button will be ok.

**Comment by KomachiSion** [comment]
Do not need to download pdf and send email.

**Comment by pengzhengfa** [comment]
@KomachiSion I've configured it locally, but I still can't

**Comment by KomachiSion** [comment]
> @KomachiSion I've configured it locally, but I still can't

I don't know how config you do, I recommend you close this PR and delete you local branch.

re-checkout a branch from develop and do commit with your new user and email again.

From my view, you config locally but re-commit and push, so the git log still is old username.

**Pull request closed without merging**